tlatools/org.lamport.tlatools/src/tla2sany/drivers/SANY.java

```diff
@@ -77,6 +77,10 @@
     // true <=> statistics about builtin operator usage 
     //          should be reported
 
+  private static boolean strictErrorCodes   = false;
+    // true <=> error level should be reported as the tools'
```

Comment:
Please bump SANY's version number and moddate.

Reply:
Done. I believe you meant the constants towards the top of the file. Let me know if I misunderstood.